libbeat/beat/event.go

```diff
@@ -47,28 +58,44 @@ type Event struct {
 }
 
 var (
-	errNoTimestamp = errors.New("value is no timestamp")
-	errNoMapStr    = errors.New("value is no map[string]interface{} type")
+	ErrValueNotTimestamp = errors.New("value is not a timestamp")
+	ErrValueNotMapStr    = errors.New("value is not `mapstr.M` or `map[string]interface{}` type")
```

Comment:
nit:
We can use `any` here instead of `interface{}`
```suggestion
	ErrValueNotMapStr    = errors.New("value is not `mapstr.M` or `map[string]any` type")
```

Reply:
I'm yet to see `any` used in any of the processors or the pipeline code. Everything is still `map[string]inteface{}` when you Unmarshal.

Reply:
I think the right thing to do would be to have a separate PR after running `gofmt -r 'interface{} -> any' -w *.go` on the entire codebase. So, we keep consistency. The consistent way at the moment is using `interface{}` for now.